Documentation/feature-band-source-building.md

```diff
@@ -0,0 +1,893 @@
+    class SBPrev2xx prev2xx
+    class SBCurrent1xx curr1xx
+    class SBCurrent2xx curr2xx
+```
+
+### 3xx Band Initial Release (N.0.300)
+
+For the initial 3xx release. The same pattern applies to 4xx band initial
+release (N.0.400), which would depend on the latest 3xx release instead of 2xx.
+
+Required inputs:
+
+- source-built artifacts from the current 1xx release
+- source-built SDK and artifacts from the previous 2xx release
+
+```bash
+git clone -b <3xx-release-branch> https://github.com/dotnet/dotnet.git
+cd dotnet
+
+# Build the SDK
+./build.sh --source-only \
+  --with-sdk /path/to/previous-source-built-2xx/sdk \
+  --with-packages /path/to/previous-source-built-2xx/artifacts \
+  --with-shared-components /path/to/current-source-built-1xx/artifacts
+
+# Final source-built outputs available in artifacts/x64/Release/
+```
+
+```mermaid
+flowchart LR
+    subgraph SBPrev2xx["Previous 2xx SB Release"]
+        Prev_2xx_SDK[Source-built 2xx SDK]
+        Prev_2xx_Art[Source-built 2xx Artifacts]
+    end
+    
+    subgraph SBCurr1xx["Current 1xx SB Release"]
+        Curr_1xx_Art[Source-built 1xx artifacts]
+    end
+    
+    subgraph SBCurr3xx["Current 3xx SB Release"]
+        Build((Build Process))
+        SDK_3xx[New 3xx SDK]
+        Art_3xx[New 3xx Artifacts]
+    end
+    
+    Prev_2xx_SDK -.-> Build
+    Prev_2xx_Art -.->|PSB| Build
+    Curr_1xx_Art -.->|shared components| Build
+    Build --> SDK_3xx
+    Build --> Art_3xx
+    
+    classDef prev2xx fill:#c98989,stroke:#666,stroke-width:2px,color:#000
+    classDef curr1xx fill:#5bb75b,stroke:#666,stroke-width:2px,color:#000
+    classDef curr3xx fill:#2d8b2d,stroke:#666,stroke-width:2px,color:#000
+    class SBPrev2xx prev2xx
+    class SBCurr1xx curr1xx
+    class SBCurr3xx curr3xx
+```
+
+### 3xx Band Bootstrap (N.0.300+)
+
+For 3xx releases that require bootstrap (two-stage process). The same pattern
+applies to 4xx band bootstrap.
+
+Required inputs:
+
+- source-built artifacts from the current 1xx release
+
+```bash
+git clone -b <3xx-release-branch> https://github.com/dotnet/dotnet.git
+cd dotnet
+
+# ===
+# Stage 1: Build with Microsoft artifacts
+# ===
+
+# Downloads Microsoft-built artifacts and Microsoft SDK
+./prep-source-build.sh
+
+# Build the SDK referencing assets from current 1xx release
+./build.sh --source-only --with-shared-components /path/to/source-built-1xx/artifacts
+
+# Extract and store the built 3xx SDK and artifacts
+tar -ozxf artifacts/assets/Release/dotnet-sdk-*-tar.gz \
+  -C /tmp/dotnet/sdk
+tar -ozxf artifacts/assets/Release/Private.SourceBuilt.Artifacts.*.tar.gz \
+  -C /tmp/dotnet/artifacts
+
+# ===
+# Stage 2: Rebuild using stage 1 outputs
+# ===
+
+./build.sh --source-only \
+  --with-sdk /tmp/dotnet/sdk \
+  --with-packages /tmp/dotnet/artifacts \
+  --with-shared-components /tmp/dotnet/artifacts
+
+# Final source-built outputs available in artifacts/x64/Release/
+```
+
+```mermaid
```

Comment:
In the graph I think it would be useful to call out the Previous MS Release is for 2xx artifacts.  A similar comment applies to `2xx Band Bootstrap (N.0.200+)`

Reply:
You made this comment before. See my response on that. https://github.com/dotnet/source-build/pull/5373#discussion_r2434078532